lib/js/belt_MapInt.js

```diff
@@ -288,14 +288,15 @@ exports.get = get;
 exports.getUndefined = getUndefined;
 exports.getWithDefault = getWithDefault;
 exports.getExn = getExn;
+exports.checkInvariantInternal = checkInvariantInternal;
 exports.remove = remove;
 exports.removeMany = removeMany;
 exports.set = set;
 exports.updateU = updateU;
 exports.update = update;
-exports.mergeArray = mergeArray;
```

Comment:
Question: does it make sense to order the exports in bucklescript to reduce the diff? The current order is restricted by the declaration in mli

Reply:
Makes sense. I guess it does not affect the semantics.